db/migrate/20220228173251_remove_drpm_from_ignorable_content.rb

```diff
@@ -1,12 +1,16 @@
 class RemoveDrpmFromIgnorableContent < ActiveRecord::Migration[6.0]
   def up
-    Katello::RootRepository.select { |r| r&.ignorable_content&.include? "drpm" }.each do |root|
-      root.ignorable_content = root.ignorable_content - ["drpm"]
+    Katello::RootRepository.select { |r| !r&.ignorable_content&.blank? }.each do |root|
+      if root&.ignorable_content && root&.ignorable_content&.include?("srpm")
```

Comment:
You don't need the first check if you use the `&`s :wink: 

```suggestion
      if root&.ignorable_content&.include?("srpm")
```